### ISSUE TITLE
schroot failed

### ISSUE THREAD

**Issue by colapsys** [neutrinolabs/pulseaudio-module-xrdp#115]
hello,

i`m using DragonOS Focal (20.04) - Lubuntu based x86_64 operating system
in order to have sound via xrdp session i try to install pulse audio but i have this issue: 

- Creating jammy build root. Log file in /var/tmp/pa-build-lab01-debootstrap.log
- Creating schroot config file /etc/schroot/chroot.d/pa-build-lab01.conf
- Copying /etc/apt/sources.list to the root
- Copying /etc/apt/sources.list.d/gnuradio-ubuntu-gnuradio-releases-jammy.list to the root
- Copying /etc/apt/sources.list.d/softwareradiosystems-ubuntu-srsran-jammy.list to the root
- Copying /etc/apt/sources.list.d/kismet.list to the root
- Copying /etc/apt/sources.list.d/ondrej-ubuntu-php-jammy.list to the root
- Copying /etc/apt/sources.list.d/mozillateam-ubuntu-ppa-jammy.list to the root
- Copying /etc/apt/sources.list.d/osmocom-latest.list to the root
- Copying /etc/apt/sources.list.d/focalx.list to the root
- Copying /etc/apt/sources.list.d/deadsnakes-ubuntu-ppa-jammy.list to the root
- Creating the build directory /build
- Copying the wrapped script to the build directory
- Building PA sources. Log file in /var/tmp/pa-build-lab01-schroot.log
** schroot failed. Check log file /var/tmp/pa-build-lab01-schroot.log

here down you can find part of the schroot.log errors.

Get:10 http://archive.ubuntu.com/ubuntu jammy/main Translation-en [510 kB]
Err:4 https://ppa.launchpadcontent.net/deadsnakes/ppa/ubuntu jammy InRelease
  The following signatures couldn't be verified because the public key is not available: NO_PUBKEY BA6932366A755776
Err:5 https://ppa.launchpadcontent.net/mozillateam/ppa/ubuntu jammy InRelease
  The following signatures couldn't be verified because the public key is not available: NO_PUBKEY AEBDF4819BE21867 NO_PUBKEY 9BDB3D89CE49EC21
Get:11 http://archive.ubuntu.com/ubuntu jammy/restricted amd64 Packages [129 kB]
Get:12 http://archive.ubuntu.com/ubuntu jammy/restricted Translation-en [18.6 kB]
Get:13 http://archive.ubuntu.com/ubuntu jammy/universe amd64 Packages [14.1 MB]
Err:9 https://www.kismetwireless.net/repos/apt/git/jammy jammy InRelease
  The following signatures couldn't be verified because the public key is not available: NO_PUBKEY 5345B8BF43403B93
Get:14 http://archive.ubuntu.com/ubuntu jammy/universe Translation-en [5652 kB]

please help !

thx.

**Comment by matt335672** [comment]
Try this for starters, which works on other focal derivatives:-

```
./install_pulseaudio_sources_apt_wrapper.sh --suite=focal
```

**Comment by colapsys** [comment]
thx for the hint, sadly didn`t work 

** schroot failed. Check log file /var/tmp/pa-build-lab01-schroot.log

should i somehow start from the beginning ?  when i tried your solution ,i got asked to delete
** Remove old build root /var/lib/pa-build/lab01 before running this script

and i did, but it is enough ?

i`m opened to suggestions.

**Comment by matt335672** [comment]
Paste the whole log and we'll look at it

**Comment by colapsys** [comment]
here it is the full log 

```
Get:1 http://deb.anydesk.com all InRelease [5588 B]
Get:2 http://archive.canonical.com/ubuntu jammy InRelease [11.4 kB]
Get:3 https://alphafox02.github.io/focalx_ppa ./ InRelease [2123 B]
Get:4 http://archive.ubuntu.com/ubuntu jammy InRelease [270 kB]
Err:1 http://deb.anydesk.com all InRelease
  The following signatures couldn't be verified because the public key is not available: NO_PUBKEY 18DF3741CDFFDE29
Get:5 https://www.kismetwireless.net/repos/apt/git/jammy jammy InRelease [13.9 kB]
Err:3 https://alphafox02.github.io/focalx_ppa ./ InRelease
  The following signatures couldn't be verified because the public key is not available: NO_PUBKEY 22A41D13482FC88D
Err:5 https://www.kismetwireless.net/repos/apt/git/jammy jammy InRelease
  The following signatures couldn't be verified because the public key is not available: NO_PUBKEY 5345B8BF43403B93
Get:6 https://ppa.launchpadcontent.net/deadsnakes/ppa/ubuntu jammy InRelease [18.1 kB]
Get:7 https://ppa.launchpadcontent.net/mozillateam/ppa/ubuntu jammy InRelease [24.6 kB]
Err:6 https://ppa.launchpadcontent.net/deadsnakes/ppa/ubuntu jammy InRelease
  The following signatures couldn't be verified because the public key is not available: NO_PUBKEY BA6932366A755776
Err:7 https://ppa.launchpadcontent.net/mozillateam/ppa/ubuntu jammy InRelease
  The following signatures couldn't be verified because the public key is not available: NO_PUBKEY AEBDF4819BE21867 NO_PUBKEY 9BDB3D89CE49EC21
Get:8 http://archive.ubuntu.com/ubuntu jammy-backports InRelease [127 kB]
Get:9 http://archive.ubuntu.com/ubuntu jammy-security InRelease [129 kB]
Get:10 http://archive.ubuntu.com/ubuntu jammy-updates InRelease [128 kB]
Get:11 http://archive.ubuntu.com/ubuntu jammy/main amd64 Packages [1395 kB]
Get:12 http://archive.ubuntu.com/ubuntu jammy/main Translation-en [510 kB]
Get:13 http://archive.ubuntu.com/ubuntu jammy/restricted amd64 Packages [129 kB]
Get:14 http://archive.ubuntu.com/ubuntu jammy/restricted Translation-en [18.6 kB]
Get:15 http://archive.ubuntu.com/ubuntu jammy/universe amd64 Packages [14.1 MB]
Get:16 http://archive.ubuntu.com/ubuntu jammy/universe Translation-en [5652 kB]
Get:17 http://archive.ubuntu.com/ubuntu jammy/multiverse amd64 Packages [217 kB]
Get:18 http://archive.ubuntu.com/ubuntu jammy/multiverse Translation-en [112 kB]
Get:19 http://archive.ubuntu.com/ubuntu jammy-backports/main amd64 Packages [67.1 kB]
Get:20 http://archive.ubuntu.com/ubuntu jammy-backports/main Translation-en [11.0 kB]
Get:21 http://archive.ubuntu.com/ubuntu jammy-backports/universe amd64 Packages [27.2 kB]
Get:22 http://archive.ubuntu.com/ubuntu jammy-backports/universe Translation-en [16.3 kB]
Get:23 http://archive.ubuntu.com/ubuntu jammy-security/main amd64 Packages [1506 kB]
Get:24 http://archive.ubuntu.com/ubuntu jammy-security/main Translation-en [257 kB]
Get:25 http://archive.ubuntu.com/ubuntu jammy-security/restricted amd64 Packages [1919 kB]
Get:26 http://archive.ubuntu.com/ubuntu jammy-security/restricted Translation-en [326 kB]
Get:27 http://archive.ubuntu.com/ubuntu jammy-security/universe amd64 Packages [857 kB]
Get:28 http://archive.ubuntu.com/ubuntu jammy-security/universe Translation-en [166 kB]
Get:29 http://archive.ubuntu.com/ubuntu jammy-security/multiverse amd64 Packages [37.2 kB]
Get:30 http://archive.ubuntu.com/ubuntu jammy-security/multiverse Translation-en [7588 B]
Get:31 http://archive.ubuntu.com/ubuntu jammy-updates/main amd64 Packages [1722 kB]
Get:32 http://archive.ubuntu.com/ubuntu jammy-updates/main Translation-en [317 kB]
Get:33 http://archive.ubuntu.com/ubuntu jammy-updates/restricted amd64 Packages [1977 kB]
Get:34 http://archive.ubuntu.com/ubuntu jammy-updates/restricted Translation-en [336 kB]
Get:35 http://archive.ubuntu.com/ubuntu jammy-updates/universe amd64 Packages [1086 kB]
Get:36 http://archive.ubuntu.com/ubuntu jammy-updates/universe Translation-en [251 kB]
Get:37 http://archive.ubuntu.com/ubuntu jammy-updates/multiverse amd64 Packages [43.0 kB]
Get:38 http://archive.ubuntu.com/ubuntu jammy-updates/multiverse Translation-en [10.7 kB]
Reading package lists...
W: GPG error: http://deb.anydesk.com all InRelease: The following signatures couldn't be verified because the public key is not available: NO_PUBKEY 18DF3741CDFFDE29
E: The repository 'http://deb.anydesk.com all InRelease' is not signed.
W: GPG error: https://alphafox02.github.io/focalx_ppa ./ InRelease: The following signatures couldn't be verified because the public key is not available: NO_PUBKEY 22A41D13482FC88D
E: The repository 'https://alphafox02.github.io/focalx_ppa ./ InRelease' is not signed.
W: GPG error: https://www.kismetwireless.net/repos/apt/git/jammy jammy InRelease: The following signatures couldn't be verified because the public key is not available: NO_PUBKEY 5345B8BF43403B93
E: The repository 'https://www.kismetwireless.net/repos/apt/git/jammy jammy InRelease' is not signed.
W: GPG error: https://ppa.launchpadcontent.net/deadsnakes/ppa/ubuntu jammy InRelease: The following signatures couldn't be verified because the public key is not available: NO_PUBKEY BA6932366A755776
E: The repository 'https://ppa.launchpadcontent.net/deadsnakes/ppa/ubuntu jammy InRelease' is not signed.
W: GPG error: https://ppa.launchpadcontent.net/mozillateam/ppa/ubuntu jammy InRelease: The following signatures couldn't be verified because the public key is not available: NO_PUBKEY AEBDF4819BE21867 NO_PUBKEY 9BDB3D89CE49EC21
E: The repository 'https://ppa.launchpadcontent.net/mozillateam/ppa/ubuntu jammy InRelease' is not signed.
Reading package lists...
Building dependency tree...
The following additional packages will be installed:
  krb5-locales libc-bin libc6 libgssapi-krb5-2 libk5crypto3 libkeyutils1
  libkrb5-3 libkrb5support0 libnsl2 libnss-nis libnss-nisplus libselinux1
  libssl3 libtirpc-common libtirpc3 locales
Suggested packages:
  manpages glibc-doc krb5-doc krb5-user
The following NEW packages will be installed:
  krb5-locales libgssapi-krb5-2 libk5crypto3 libkeyutils1 libkrb5-3
  libkrb5support0 libnsl2 libnss-nis libnss-nisplus libssl3 libtirpc-common
  libtirpc3
The following packages will be upgraded:
  libc-bin libc6 libselinux1 locales lsb-release sudo
6 upgraded, 12 newly installed, 0 to remove and 182 not upgraded.
Need to get 11.8 MB of archives.
After this operation, 8063 kB of additional disk space will be used.
Get:1 http://archive.ubuntu.com/ubuntu jammy-security/main amd64 locales all 2.35-0ubuntu3.8 [4245 kB]
Get:2 http://archive.ubuntu.com/ubuntu jammy-security/main amd64 libc6 amd64 2.35-0ubuntu3.8 [3235 kB]
Get:3 http://archive.ubuntu.com/ubuntu jammy-security/main amd64 libc-bin amd64 2.35-0ubuntu3.8 [706 kB]
Get:4 http://archive.ubuntu.com/ubuntu jammy/main amd64 libselinux1 amd64 3.3-1build2 [74.6 kB]
Get:5 http://archive.ubuntu.com/ubuntu jammy-security/main amd64 libkrb5support0 amd64 1.19.2-2ubuntu0.3 [32.2 kB]
Get:6 http://archive.ubuntu.com/ubuntu jammy-security/main amd64 libk5crypto3 amd64 1.19.2-2ubuntu0.3 [86.3 kB]
Get:7 http://archive.ubuntu.com/ubuntu jammy/main amd64 libkeyutils1 amd64 1.6.1-2ubuntu3 [10.4 kB]
Get:8 http://archive.ubuntu.com/ubuntu jammy-security/main amd64 libssl3 amd64 3.0.2-0ubuntu1.15 [1905 kB]
Get:9 http://archive.ubuntu.com/ubuntu jammy-security/main amd64 libkrb5-3 amd64 1.19.2-2ubuntu0.3 [356 kB]
Get:10 http://archive.ubuntu.com/ubuntu jammy-security/main amd64 libgssapi-krb5-2 amd64 1.19.2-2ubuntu0.3 [144 kB]
Get:11 http://archive.ubuntu.com/ubuntu jammy-security/main amd64 libtirpc-common all 1.3.2-2ubuntu0.1 [7766 B]
Get:12 http://archive.ubuntu.com/ubuntu jammy-security/main amd64 libtirpc3 amd64 1.3.2-2ubuntu0.1 [82.3 kB]
Get:13 http://archive.ubuntu.com/ubuntu jammy/main amd64 libnsl2 amd64 1.3.0-2build2 [42.3 kB]
Get:14 http://archive.ubuntu.com/ubuntu jammy/main amd64 lsb-release all 11.1.0ubuntu4 [10.8 kB]
Get:15 http://archive.ubuntu.com/ubuntu jammy-security/main amd64 sudo amd64 1.9.9-1ubuntu2.4 [821 kB]
Get:16 http://archive.ubuntu.com/ubuntu jammy-security/main amd64 krb5-locales all 1.19.2-2ubuntu0.3 [11.7 kB]
Get:17 http://archive.ubuntu.com/ubuntu jammy/main amd64 libnss-nis amd64 3.1-0ubuntu6 [27.3 kB]
Get:18 http://archive.ubuntu.com/ubuntu jammy/main amd64 libnss-nisplus amd64 1.3-0ubuntu6 [22.9 kB]
Preconfiguring packages ...
Fetched 11.8 MB in 2s (6864 kB/s)
(Reading database ... 
(Reading database ... 5%
(Reading database ... 10%
(Reading database ... 15%
(Reading database ... 20%
(Reading database ... 25%
(Reading database ... 30%
(Reading database ... 35%
(Reading database ... 40%
(Reading database ... 45%
(Reading database ... 50%
(Reading database ... 55%
(Reading database ... 60%
(Reading database ... 65%
(Reading database ... 70%
(Reading database ... 75%
(Reading database ... 80%
(Reading database ... 85%
(Reading database ... 90%
(Reading database ... 95%
(Reading database ... 100%
(Reading database ... 11356 files and directories currently installed.)
Preparing to unpack .../locales_2.35-0ubuntu3.8_all.deb ...
Unpacking locales (2.35-0ubuntu3.8) over (2.31-0ubuntu9) ...
Preparing to unpack .../libc6_2.35-0ubuntu3.8_amd64.deb ...
Unpacking libc6:amd64 (2.35-0ubuntu3.8) over (2.31-0ubuntu9) ...
Setting up libc6:amd64 (2.35-0ubuntu3.8) ...
(Reading database ... 
(Reading database ... 5%
(Reading database ... 10%
(Reading database ... 15%
(Reading database ... 20%
(Reading database ... 25%
(Reading database ... 30%
(Reading database ... 35%
(Reading database ... 40%
(Reading database ... 45%
(Reading database ... 50%
(Reading database ... 55%
(Reading database ... 60%
(Reading database ... 65%
(Reading database ... 70%
(Reading database ... 75%
(Reading database ... 80%
(Reading database ... 85%
(Reading database ... 90%
(Reading database ... 95%
(Reading database ... 100%
(Reading database ... 11338 files and directories currently installed.)
Preparing to unpack .../libc-bin_2.35-0ubuntu3.8_amd64.deb ...
Unpacking libc-bin (2.35-0ubuntu3.8) over (2.31-0ubuntu9) ...
Setting up libc-bin (2.35-0ubuntu3.8) ...
(Reading database ... 
(Reading database ... 5%
(Reading database ... 10%
(Reading database ... 15%
(Reading database ... 20%
(Reading database ... 25%
(Reading database ... 30%
(Reading database ... 35%
(Reading database ... 40%
(Reading database ... 45%
(Reading database ... 50%
(Reading database ... 55%
(Reading database ... 60%
(Reading database ... 65%
(Reading database ... 70%
(Reading database ... 75%
(Reading database ... 80%
(Reading database ... 85%
(Reading database ... 90%
(Reading database ... 95%
(Reading database ... 100%
(Reading database ... 11336 files and directories currently installed.)
Preparing to unpack .../libselinux1_3.3-1build2_amd64.deb ...
Unpacking libselinux1:amd64 (3.3-1build2) over (3.0-1build2) ...
Setting up libselinux1:amd64 (3.3-1build2) ...
Selecting previously unselected package libkrb5support0:amd64.
(Reading database ... 
(Reading database ... 5%
(Reading database ... 10%
(Reading database ... 15%
(Reading database ... 20%
(Reading database ... 25%
(Reading database ... 30%
(Reading database ... 35%
(Reading database ... 40%
(Reading database ... 45%
(Reading database ... 50%
(Reading database ... 55%
(Reading database ... 60%
(Reading database ... 65%
(Reading database ... 70%
(Reading database ... 75%
(Reading database ... 80%
(Reading database ... 85%
(Reading database ... 90%
(Reading database ... 95%
(Reading database ... 100%
(Reading database ... 11336 files and directories currently installed.)
Preparing to unpack .../00-libkrb5support0_1.19.2-2ubuntu0.3_amd64.deb ...
Unpacking libkrb5support0:amd64 (1.19.2-2ubuntu0.3) ...
Selecting previously unselected package libk5crypto3:amd64.
Preparing to unpack .../01-libk5crypto3_1.19.2-2ubuntu0.3_amd64.deb ...
Unpacking libk5crypto3:amd64 (1.19.2-2ubuntu0.3) ...
Selecting previously unselected package libkeyutils1:amd64.
Preparing to unpack .../02-libkeyutils1_1.6.1-2ubuntu3_amd64.deb ...
Unpacking libkeyutils1:amd64 (1.6.1-2ubuntu3) ...
Selecting previously unselected package libssl3:amd64.
Preparing to unpack .../03-libssl3_3.0.2-0ubuntu1.15_amd64.deb ...
Unpacking libssl3:amd64 (3.0.2-0ubuntu1.15) ...
Selecting previously unselected package libkrb5-3:amd64.
Preparing to unpack .../04-libkrb5-3_1.19.2-2ubuntu0.3_amd64.deb ...
Unpacking libkrb5-3:amd64 (1.19.2-2ubuntu0.3) ...
Selecting previously unselected package libgssapi-krb5-2:amd64.
Preparing to unpack .../05-libgssapi-krb5-2_1.19.2-2ubuntu0.3_amd64.deb ...
Unpacking libgssapi-krb5-2:amd64 (1.19.2-2ubuntu0.3) ...
Selecting previously unselected package libtirpc-common.
Preparing to unpack .../06-libtirpc-common_1.3.2-2ubuntu0.1_all.deb ...
Unpacking libtirpc-common (1.3.2-2ubuntu0.1) ...
Selecting previously unselected package libtirpc3:amd64.
Preparing to unpack .../07-libtirpc3_1.3.2-2ubuntu0.1_amd64.deb ...
Unpacking libtirpc3:amd64 (1.3.2-2ubuntu0.1) ...
Selecting previously unselected package libnsl2:amd64.
Preparing to unpack .../08-libnsl2_1.3.0-2build2_amd64.deb ...
Unpacking libnsl2:amd64 (1.3.0-2build2) ...
Preparing to unpack .../09-lsb-release_11.1.0ubuntu4_all.deb ...
Unpacking lsb-release (11.1.0ubuntu4) over (11.1.0ubuntu2) ...
Preparing to unpack .../10-sudo_1.9.9-1ubuntu2.4_amd64.deb ...
Unpacking sudo (1.9.9-1ubuntu2.4) over (1.8.31-1ubuntu1) ...
Selecting previously unselected package krb5-locales.
Preparing to unpack .../11-krb5-locales_1.19.2-2ubuntu0.3_all.deb ...
Unpacking krb5-locales (1.19.2-2ubuntu0.3) ...
Selecting previously unselected package libnss-nis:amd64.
Preparing to unpack .../12-libnss-nis_3.1-0ubuntu6_amd64.deb ...
Unpacking libnss-nis:amd64 (3.1-0ubuntu6) ...
Selecting previously unselected package libnss-nisplus:amd64.
Preparing to unpack .../13-libnss-nisplus_1.3-0ubuntu6_amd64.deb ...
Unpacking libnss-nisplus:amd64 (1.3-0ubuntu6) ...
Setting up libkeyutils1:amd64 (1.6.1-2ubuntu3) ...
Setting up libtirpc-common (1.3.2-2ubuntu0.1) ...
Setting up libssl3:amd64 (3.0.2-0ubuntu1.15) ...
Setting up krb5-locales (1.19.2-2ubuntu0.3) ...
Setting up locales (2.35-0ubuntu3.8) ...
Installing new version of config file /etc/locale.alias ...
Generating locales (this might take a while)...
Generation complete.
Setting up libkrb5support0:amd64 (1.19.2-2ubuntu0.3) ...
Setting up sudo (1.9.9-1ubuntu2.4) ...
Installing new version of config file /etc/pam.d/sudo ...
Installing new version of config file /etc/sudoers ...
Installing new version of config file /etc/sudoers.d/README ...
Setting up libk5crypto3:amd64 (1.19.2-2ubuntu0.3) ...
Setting up libkrb5-3:amd64 (1.19.2-2ubuntu0.3) ...
Setting up lsb-release (11.1.0ubuntu4) ...
Setting up libgssapi-krb5-2:amd64 (1.19.2-2ubuntu0.3) ...
Setting up libtirpc3:amd64 (1.3.2-2ubuntu0.1) ...
Setting up libnsl2:amd64 (1.3.0-2build2) ...
Setting up libnss-nisplus:amd64 (1.3-0ubuntu6) ...
Setting up libnss-nis:amd64 (3.1-0ubuntu6) ...
Processing triggers for libc-bin (2.35-0ubuntu3.8) ...
Building for : Ubuntu-20.04 (focal)
- Adding source repositories
Reading package lists...
Reading package lists...
E: You must put some 'deb-src' URIs in your sources.list
```

**Comment by matt335672** [comment]
Looks like #103

Are you using the latest version of the scripts, or a release? Try with the latest version of the git sources.
